modules/virify_qc.nf

```diff
@@ -5,7 +5,7 @@ process VIRIFY_QC {
 
     // FIXME: this module doesn't produce a valid GFF
     // gt gff3validator: error: line 1 in file "virify_hq.gff" does not begin with "##gff-version" or "##gff-version"
```

Comment:
This is an intermediary gff-like file with no headers that doesn't need validation as it is parsed internally by one of our python scripts